templates/eap-cd-postgresql-persistent-s2i.json

```diff
@@ -0,0 +1,881 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,javaee,java,jboss",
+            "version": "1.4.9",
```

Comment:
I just noticed this when comparing this one to the 7.1 one but I suspect the same issue may apply to all the others. In 7.1 these "versions" are 1.4.10

https://github.com/jboss-openshift/application-templates/commit/09604c86eb78362a602944d23e7f3d8439a8f1f4

Reply:
@bstansberry Thanks for both noticing that and reminding me. 

@rcernich How are we planning to manage these versions? Do they need to match the version tagged with the other templates? (Those are currently 1.4.10.)